devices/Vl6180X/samples/Vl6180x.sample/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("CSharp.BlankApplication")]
```

Comment:
I know it's the sample but always nice to adjust ;-)
and clean the file for what's not needed

Reply:
done